planning/behavior_path_planner/src/scene_module/pull_out/pull_out_module.cpp

```diff
@@ -605,38 +605,32 @@ bool PullOutModule::hasFinishedCurrentPath()
 
 TurnSignalInfo PullOutModule::calcTurnSignalInfo(const Pose start_pose, const Pose end_pose) const
 {
-  TurnSignalInfo turn_signal;
+  TurnSignalInfo turn_signal{};  // output
 
-  // turn hazard light when backward driving
+  const Pose & current_pose = planner_data_->self_pose->pose;
+
+  // turn on hazard light when backward driving
```

Comment:
- add else
- desired start point = start point of pull out

Reply:
fixed in https://github.com/tier4/autoware.universe/pull/138/commits/09b7e451b7e717680fa601e873e240acb5a2ab91